pipeline_dp/dp_computations.py

```diff
@@ -529,6 +536,44 @@ def describe(self) -> str:
                 f"l2_sensitivity={self.sensitivity}")
 
 
+class MeanMechanism:
+    """Computes DP mean.
+
+    It computes DP mean as a ratio of DP sum and DP count. For improving
+    utility the normalization to [min_value, max_value] is performed.
```

Comment:
Isn't it rather normalization to `mid_value`?

Reply:
Thanks, I've updated the docstring